data/crac/crac-impl/src/test/java/com/farao_community/farao/data/crac_impl/json/CracImportExportTest.java

```diff
@@ -117,18 +112,12 @@ public void cracTest() {
             Stream.of(simpleCrac.getNetworkElement("pst"), simpleCrac.addNetworkElement("pst2")).collect(Collectors.toSet())
         ));
 
-        ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
-        CracExporters.exportCrac(simpleCrac, "Json", outputStream);
-        try (ByteArrayInputStream inputStream = new ByteArrayInputStream(outputStream.toByteArray())) {
-            Crac crac = CracImporters.importCrac("unknown.json", inputStream);
-            assertEquals(5, crac.getNetworkElements().size());
-            assertEquals(2, crac.getInstants().size());
-            assertEquals(2, crac.getContingencies().size());
-            assertEquals(3, crac.getCnecs().size());
-            assertEquals(2, crac.getRangeActions().size());
-            assertEquals(2, crac.getNetworkActions().size());
-        } catch (IOException e) {
-            throw new UncheckedIOException(e);
-        }
+        Crac crac = roundTrip(simpleCrac, SimpleCrac.class);
+        assertEquals(5, crac.getNetworkElements().size());
+        assertEquals(2, crac.getInstants().size());
+        assertEquals(2, crac.getContingencies().size());
+        assertEquals(3, crac.getCnecs().size());
+        assertEquals(2, crac.getRangeActions().size());
+        assertEquals(2, crac.getNetworkActions().size());
```

Comment:
maybe also add an import test, thus we can monitor the changes in the JSON format that could broke some FARAO uses